uberfire-structure/uberfire-structure-client/src/main/resources/org/guvnor/structure/client/resources/i18n/CommonConstants.properties

```diff
@@ -9,13 +9,13 @@ RepositoryActionRead=Read
 RepositoryActionUpdate=Update
 RepositoryActionDelete=Delete
 RepositoryActionCreate=Create
-OrganizationalUnitsNode=Organizational Units
-OrganizationalUnitsHelp=Sets who can Create, Update or Delete organizational units from the Organizational Unit section at the Administration perspective. Set also what organizational units are visible in the Project Explorer at the Project Authoring perspective.
-OrganizationalUnitResource=Organizational Unit
-OrganizationalUnitActionRead=Read
-OrganizationalUnitActionUpdate=Update
-OrganizationalUnitActionDelete=Delete
-OrganizationalUnitActionCreate=Create
+SpacesNode=Spaces
+SpacesHelp=Sets who can Create, Update or Delete spaces from the Spaces section at the Administration perspective. Set also what spaces are visible in the Project Explorer at the Project Authoring perspective.
+SpaceResource=Organizational Unit
```

Comment:
@adrielparedes I'm too late to the party, but looks like you've forgotten "Organizational Unit" here?